### ISSUE TITLE
may you upadte a AVA dataset tutorial for newbie?

### ISSUE THREAD

**Issue by Biel-Hammer** [facebookresearch/pytorchvideo#193]
## 🚀 Feature

NOTE: Please look at the existing list of Issues tagged with the label ['enhancement`](https://github.com/facebookresearch/pytorchvideo/issues?q=label%3Aenhancement). **Only open a new issue if you do not see your feature request there**.

## Motivation

**Your tutorial for video classification training or finetuning is just for KineticsDat, may you upadte a AVA dataset tutorial for newbie?**

## Pitch

NOTE: we only consider adding new features if they are useful for many users.

**Comment by kalyanvasudev** [comment]
Hi We don't have plans to support this Pytorchvideo anytime soon. I'd encourage users to the AVA benchmarks in pyslowfast repo to do this - https://github.com/facebookresearch/slowfast
